x-pack/plugins/data_visualizer/public/application/index_data_visualizer/data_loader/data_loader.ts

```diff
@@ -31,8 +30,6 @@ export class DataLoader {
     toastNotifications: CoreSetup['notifications']['toasts']
   ) {
     this._indexPattern = indexPattern;
-    this._runtimeMappings = this._indexPattern.getComputedFields()
```

Comment:
`indexPattern.getComputedFields()
      .runtimeFields` is not a mutable reference anymore.